plugins/module_utils/turbo/module.py

```diff
@@ -147,7 +147,8 @@ def run_on_daemon(self):
             ansiblez_path,
             json.dumps(args),
         ]
-        _socket.send(json.dumps(data).encode())
+        _socket.sendall(json.dumps(data).encode())
+        _socket.shutdown(socket.SHUT_WR)
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Done, thanks!